Header header
uint8[] img
int32 col
int32 row
int8 channels
int8 compressionFlag
int32 compressionQuality
string compressName
